src/main/resources/spring/camel-context.xml

```diff
@@ -88,14 +118,16 @@
         <rest path="/mappings">
             <description>Mapping REST API</description>
             <get uri="/flag-assessment">
+                <param name="offset" type="query" defaultValue="0" required="false" description="Define the page requested" />
+                <param name="limit" type="query" defaultValue="{{pagination.flag-assessment.limit.default}}" required="false" description="Define the size of the requested page" />
                 <route id="mappings-flag-assessment-findAll">
                     <!-- RBAC check start-->
                     <setHeader headerName="${type:org.jboss.xavier.integrations.rbac.RBACRouteBuilder.RBAC_ENDPOINT_RESOURCE_NAME}"><constant>mappings</constant></setHeader>
                     <setHeader headerName="${type:org.jboss.xavier.integrations.rbac.RBACRouteBuilder.RBAC_ENDPOINT_RESOURCE_PERMISSION}"><constant>read</constant></setHeader>
                     <to uri="direct:check-rbac-permissions" />
                     <!-- RBAC check end-->
 
-                    <bean ref="flagAssessmentService" method="findAll()" />
+                    <bean ref="flagAssessmentService" method="findAll(${header.offset}, ${header.limit})" />
```

Comment:
Same as above

Reply:
I dont understand what you mean. This endpoint needs to have this pagination features as it is requested on the API guide.

Reply:
After our conversation , although these changes were done 1 week ago, I will remove them as you have just already done the same in your PR.